src/cli/commands/__tests__/lint.test.ts

```diff
@@ -17,6 +17,8 @@ const validRulesetConfigPath = resolve(__dirname, '__fixtures__/config.ruleset.y
 const invalidRulesetConfigPath = resolve(__dirname, '__fixtures__/config.ruleset.invalid.yml');
 const standardOas3RulesetPath = resolve(__dirname, '../../../rulesets/oas3/ruleset.json');
 const standardOas2RulesetPath = resolve(__dirname, '../../../rulesets/oas2/ruleset.json');
+const draftRefSpec = resolve(__dirname, './__fixtures__/draft-ref.oas2.json');
```

Comment:
I don't understand what "draft" means in this context?

Reply:
I couldn't think of anything more meaningful.
It's just an almost empty spec, hence the draft in the name, but happy to rename it.

Reply:
ahh right. Most of these fixtures are bare minimal API descriptions so no need to differentiate any how.